bigquery_etl/shredder/delete.py

```diff
@@ -274,6 +277,24 @@ def create_job(client):
             ):
                 replace_clause = BUG_1751979_MAIN_SUMMARY_V4_REPLACE_CLAUSE
                 temporary_udfs = BUG_1751979_MAIN_SUMMARY_V4_UDFS
+            # And corresponding changes for clients_daily-derived tables.
+            elif sql_table_id(target) in [
+                "moz-fx-data-shared-prod.telemetry_derived.clients_daily_v6",
+                "moz-fx-data-shared-prod.telemetry_derived.clients_daily_joined_v1",
+                "moz-fx-data-shared-prod.telemetry_derived.clients_last_seen_v6",
+                "moz-fx-data-shared-prod.telemetry_derived.clients_last_seen_joined_v1",
+            ]:
+                replace_clause = BUG_1751979_CLIENTS_DAILY_V6_REPLACE_CLAUSE
+                temporary_udfs = (
+                    BUG_1751979_MAIN_SUMMARY_V4_UDFS + BUG_1751979_MAIN_V4_UDFS
+                )
+            # And corresponding changes for search_clients_daily-derived tables.
+            elif (
+                sql_table_id(target)
+                == "moz-fx-data-shared-prod.telemetry_derived.search_clients_daily_v8"
```

Comment:
@jklukas of these only `clients_daily_v6` is actually being scrubbed. `telemetry_derived.clients_daily_joined_v1` and `telemetry_derived.clients_last_seen_joined_v1` aren't configured for deletes, and `search_clients_daily_v8` is in the `search_derived` dataset, not the `telemetry_derived` dataset.